v0.12.19 release (#79)
reddit/devvit-docs: docs/changelog.md

**Please note**: you may see features available across Devvit packages that are not documented or noted in our changelog. These are experimental features that are not stable and are subject to change, or removal, from the platform. Please use caution when testing or implementing experimental features.

## Release 0.12.19: Minor Updates
**Release Date: Apr 13, 2026**

This release includes the following updates: 

* Clarified that when `requestExpandedMode()` navigates to the same entrypoint, it may (but not always) trigger a reload.

> * Added a `post` parameter to the `shareSheet` effect to allow association with a specific post ID.

Reviewer: ```suggestion
* Added a `post` parameter to the `shareSheet` effect to allow sharing links to other posts.
```
